src/views/Home/CDKTypeTabs.tsx

```diff
@@ -91,8 +92,6 @@ export const CDKTypeTabs: FunctionComponent = () => {
 
   return (
     <Flex
-      bg="white"
-      color="blue.800"
```

Comment:
I would continue setting the `color` here and allowing inheritance to set it for the text below

Reply:
TIL!